presto-main/src/main/java/io/prestosql/sql/planner/NodePartitioningManager.java

```diff
@@ -207,7 +207,7 @@ private ConnectorBucketNodeMap getConnectorBucketNodeMap(Session session, Partit
 
```

Comment:
change commit title: `Use proper connector session in ConnectorNodePartitioningProvider`

Reply:
Title changed!   :)